public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/AlertManagerRouting.tsx

```diff
@@ -68,11 +43,12 @@ export function AlertManagerManualRouting({ alertManager }: AlertManagerManualRo
         <div className={styles.secondAlertManagerLine} />
       </Stack>
       <Stack direction="row" gap={1} alignItems="center">
-        <ContactPointSelector alertManager={alertManagerName} onSelectContactPoint={onSelectContactPoint} />
+        <ContactPointSelector alertManager={alertManagerName} />
       </Stack>
-      {selectedContactPointWithMetadata?.grafana_managed_receiver_configs && (
-        <ContactPointDetails receivers={selectedContactPointWithMetadata.grafana_managed_receiver_configs} />
-      )}
+      {/* @TODO
```

Comment:
This is a bit unclear. What details we lose?

Reply:
Yeah, because I think we have the contact point metadata, that is what we need for details? or am I missing something?